rasa/utils/plotting.py

```diff
@@ -135,13 +135,51 @@ def plot_histogram(
     colors = ["#009292", "#920000"]
     bins = [0.05 * i for i in range(1, 21)]
 
-    plt.xlim([0, 1])
-    plt.hist(hist_data, bins=bins, color=colors)
-    plt.xticks(bins)
-    plt.title(title)
-    plt.xlabel("Confidence")
-    plt.ylabel("Number of Samples")
-    plt.legend(["hits", "misses"])
+    binned_data_sets = [np.histogram(d, bins=bins)[0] for d in hist_data]
+
+    max_xlim = max(binned_data_sets[0] + binned_data_sets[1])
```

Comment:
See my response -- I think we should have one max_xlim each for the left and right sides

Reply:
Done.